tests/agent/test_agent.py

```diff
@@ -33,7 +33,7 @@
     from tests.agent.conftest import DataContextConfigTD
 
 
-@pytest.fixture(autouse=True)
```

Comment:
(Since I added a test with bad config, we no longer want to autouse this fixture)